.gitignore

```diff
@@ -58,3 +58,6 @@ data/
 # Local environment overrides
 .env
 settings.ini
+
+# l10n
+locale/
```

Comment:
I'm leaning toward keeping the locale finds in this repo, same as kuma. The only reason I can think to do a separate repo is to keep all the BC strings across different projects together, but that seems like a lot of work for a leaky abstraction.